pymc3/distributions/multivariate.py

```diff
@@ -113,7 +124,10 @@ def logp(self, value):
         delta = value - mu
         k = tau.shape[0]
 
-        result = k * tt.log(2 * np.pi) + tt.log(1. / det(tau))
+        if self.gpu_compat:
+            result = k * tt.log(2 * np.pi) + tt.log(1. / det(tau))
```

Comment:
the first term can be moved up top, then the if-clause just does -logdet or -tt.log(det(tau)